testers/testers/custom/markus_custom_tester.py

```diff
@@ -1,11 +1,12 @@
 import subprocess
-import os
+
 
 class MarkusCustomTester:
```

Comment:
Not strictly necessary, but I believe it should inherit from MarkusTester anyway.